docs/dev/building.md

```diff
@@ -22,37 +22,45 @@ git clone https://github.com/scylladb/scylla
 cd ./scylla
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py --mode=<mode>
-./tools/toolchain/dbuild ninja
+./tools/toolchain/dbuild cmake --build build
```

Comment:
This surprises me: after `configure.py`, I just run something like
```
ninja -C build {scylla,tests}:Dev
```

and indeed your update above, to `README.md`, near the `dbuild` stuff, recommends `ninja -C build scylla:RelWithDebInfo`. So this part looks at least inconsistent (even though it may perfectly well work -- I've never tried it).

Reply:
i think, `README.md` should serve as a minimal but workable solution. `building.md` can expand the topic further by providing more options. the upside of using `cmake` to drive the build is, we don't need to use the less-documented notation of `<target>:<config>` when we need to specify the build mode / config.